fasthx/main.py

```diff
@@ -17,52 +19,54 @@ def _append_to_signature(
     func: Callable[_P, _T],
     *params: inspect.Parameter,
 ) -> Callable[_P, _T]:
-    """
-    Appends the given parameters to the *end* of signature of the given function.
+    """Appends the given parameters to the *end* of signature of the given function.
 
     Notes:
+    -----
         - This method does not change the function's arguments, it only makes FastAPI's
         dependency resolution system recognize inserted parameters.
         - This is *not* a general purpose method, it is strongly recommended to only
         append keyword-only parameters that have "unique" names that are unlikely to
         be already in the function's signature.
 
     Arguments:
+    ---------
         func: The function whose signature should be extended.
         params: The parameters to add to the function's signature.
 
     Returns:
+    -------
         The received function with an extended `__signature__`.
-    """
+
+    """  # noqa: D401
```

Comment:
`pyproject.toml` contains the linter config - I'm assuming your local development environment did not load the settings from there, otherwise there wouldn't be a linter warning here. I'll update the CI to run on PRs - that would flag your changes.

Reply:
Perhaps worth putting down a contributor guideline on these things. And yeah, definitely a good idea to have CI check on these things.

Reply:
Yeah, adding more docs on how to do linting and formatting locally (`poethepoet` tasks) would be helpful, but I think following the already existing coding and documentation pattern should be possible regardless.

Th CI check is already in place - I added it not long ago.